### PR TITLE
Implement buttons on testpage

### PR DESCRIPTION
**Pull request by HajimeNoJosh**
Made a temporary input with a function startTimer(). This is only
temporary and will be cleaned up later.

Currently the button functions the way it does through a new state
isPauseButtonShowing. This state dictates whether to show the pause
button or to show the start/restart buttons. It does so in the toggle
function. It just sets pauseButtonShowing to the opposite of isActive.

### CODE REVIEW

File: src/components/TestPage/TestPage.jsx [hunk 1]
```diff
@@ -40,28 +44,47 @@ export const TestPage = () => {
...
+        {!isPauseShowing && (
+          <Button
+            onClick={restart}
+            aria-label="restart-button"
+            disabled={isActive}
+            variant="secondary"
+            iconName="restart"
+            text="Restart"
+          />
+        )}
```
Comment (paulinecat) [inline src/components/TestPage/TestPage.jsx:87]: If you want to save yourself a few lines you can do
```jsx
        {isPauseShowing ? (
          <Button { ...props } /> // Pause button
        ) : (
          <>
            <Button { ...props } /> // Play button
            <Button { ...props } /> // Restart button
          </>
        )}
```

Comment (HajimeNoJosh) [reply src/components/TestPage/TestPage.jsx:87]: Makes sense.  
```js
<>
   <Button { ...props } /> // Play button
   <Button { ...props } /> // Restart button
 </>
```

For this part ^ are we wrapping it with an empty tag or should it be a div/fragment maybe?

Comment (paulinecat) [reply src/components/TestPage/TestPage.jsx:87]: Probably a div, since we do need the block with those two buttons to be centered

Comment (HajimeNoJosh) [reply src/components/TestPage/TestPage.jsx:87]: I gotcha. We already have a div that can center the the whole block of all the buttons, but another div to wrap the two buttons could be useful for generally manipulating the buttons inside the div.

**Pull request merged**